packages/Input/stories/examples/InputExample.js

```diff
@@ -4,7 +4,19 @@ import Input from "../../src";
 const InputExample = React.forwardRef((props, ref) => {
   const [value, setValue] = React.useState(props.value || "");
 
-  return <Input {...props} onChange={e => setValue(e.target.value)} ref={ref} value={value} />;
+  return (
+    <Input
+      {...props}
+      onChange={e => {
+        setValue(e.target.value);
+      }}
+      onClear={() => {
+        setValue("");
+      }}
```

Comment:
This illustrates how a controlled component _should_ be used if it has a clear button.  It should not rely on the `onChange` prop alone.

Reply:
Would be awesome if we can avoid onClear and just dispatch an onChange event with `""` as value.

Reply:
Agreed. But was unable to get that to work.